libraries/standard/mqtt/utest/mqtt_lightweight_utest.c

```diff
@@ -0,0 +1,177 @@
+#include <stdbool.h>
```

Comment:
Please don't include stdbool.h directly as it is a C99+ header. I think you can just delete it as it is handled here
https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/development/libraries/standard/mqtt/include/mqtt_lightweight.h#L26